rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -82,6 +83,10 @@ mv %{buildroot}%{gem_instdir}/extra/smart-proxy-openscap-send.cron \
 # create spool directory
 mkdir -p %{buildroot}%{spool_dir}
 
+# create content directory and symlink it to foreman-proxy directory
+mkdir -p %{buildroot}%{content_dir}/content
+ln -sv %{buildroot}%{content_dir} %{buildroot}%{_datadir}/foreman-proxy/openscap
```

Comment:
symlink target should not be inside the buildroot, just %{content_dir}

    error: Symlink points to BuildRoot: /usr/share/foreman-proxy/openscap -> /builddir/build/BUILDROOT/rubygem-smart_proxy_openscap-0.4.1-1.git.96.6691a12.git.96.6691a12.git.96.6691a12.el7.noarch/var/lib/foreman-proxy/openscap